community/cypher/frontend-3.0/src/main/scala/org/neo4j/cypher/internal/frontend/v3_0/ast/functions/Point.scala

```diff
@@ -19,12 +19,40 @@
  */
 package org.neo4j.cypher.internal.frontend.v3_0.ast.functions
 
-import org.neo4j.cypher.internal.frontend.v3_0.ast.{Function, SimpleTypedFunction}
+import org.neo4j.cypher.internal.frontend.v3_0.ast.Expression.SemanticContext
+import org.neo4j.cypher.internal.frontend.v3_0.ast._
 import org.neo4j.cypher.internal.frontend.v3_0.symbols._
+import org.neo4j.cypher.internal.frontend.v3_0.{SemanticCheck, SemanticCheckResult, SemanticError}
 
 case object Point extends Function with SimpleTypedFunction {
   def name = "point"
 
+  override def semanticCheck(ctx: SemanticContext, invocation: FunctionInvocation) =
+    super.semanticCheck(ctx, invocation) ifOkChain  checkPointMap(invocation.args(0))
+
+  /*
+   * Checks so that the point map is properly formatted
+   */
+  protected def checkPointMap(expression: Expression): SemanticCheck = expression match {
+    //Cartesian point
+    case map: MapExpression if map.items.exists(withKey("x")) &&  map.items.exists(withKey("y")) =>
+        if (map.items.exists(withKey("crs"))) SemanticCheckResult.success
+        else SemanticError(s"A cartesian point must contain a 'crs' (coordinate reference system)", map.position)
+    //Geographic point
+    case map: MapExpression if map.items.exists(withKey("longitude")) &&  map.items.exists(withKey("latitude")) =>
```

Comment:
yeah but that isn't my fault :) 

The syntax is specified so that if using geographic coordinates the `crs` can be omitted and will default to `WGS-84`.